src/Bswup/Bit.Bswup.Demo/wwwroot/index.html

```diff
@@ -102,10 +106,10 @@
                     progressEl.style.display = 'block';
                     return console.log('installing new version:', data.version);
                 case 'installed':
-                    reloadButton.style.display = 'block';
                     console.log('new version installed:', data.version)
+                    //reloadButton.style.display = 'block';
+                    //reloadButton.onclick = data.reload;
```

Comment:
Do we need this?

Reply:
yes, for now. we need to investigate why this part of the lifecycle not working as intended.
we can remove it later.